src/index.ts

```diff
@@ -39,13 +65,22 @@ const endpoints = [
 const onEvent =
   (pageview = false) =>
   async (event: MCEvent) => {
+    const eventId = pageview ? crypto.randomUUID() : null
```

Comment:
```suggestion
```

Reply:
This is needed so that the same event ID is sent for the two different pageview endpoints.